lib/workers/repository/process/write.ts

```diff
@@ -64,6 +76,14 @@ export async function writeUpdates(
     ) {
       deductPrRemainingCount = 1;
     }
+    // istanbul ignore if
```

Comment:
This should be tested?

Reply:
I tested it manually, most probably this and other checks above will go

Reply:
ok, we should add test later